app/src/main/java/org/wikipedia/dataclient/mwapi/MwQueryPage.kt

```diff
@@ -24,7 +24,7 @@ class MwQueryPage {
     @SerialName("pageid") val pageId = 0
     @SerialName("pageprops") val pageProps: PageProps? = null
     @SerialName("entityterms") val entityTerms: EntityTerms? = null
-    @SerialName("categories") val categoriesProps: List<CategoryResponse>? = null
+    @SerialName("categories") val categories: List<CategoryResponse>? = null
```

Comment:
Do we still need the `@SerialName` if the name equals the variable name?

Reply:
haha that was my intention!